platform/src/main/resources/anchor-config-defaults.yaml

```diff
@@ -86,16 +86,16 @@ app-config:
     #     NONE: no authentication is used
     #     API_KEY: the authentication is done using an API key added to the `X-Api-Key` header.
     #     JWT_TOKEN: the authentication is done using a JWT token added to the `Authorization` header. this token is generated from the secret key.
-    authType: JWT_TOKEN
+    authType: NONE
```

Comment:
The default value for the reference server is still JWT_TOKEN though:
https://github.com/stellar/java-stellar-anchor-sdk/blob/702bbb66cf44491cc0af39223abb108a8e98e44f/anchor-reference-server/src/main/resources/anchor-reference-server.yaml#L26

Reply:
Fixed!